athena-saphana/src/main/java/com/amazonaws/athena/connectors/saphana/SaphanaQueryStringBuilder.java

```diff
@@ -152,8 +154,18 @@ public PreparedStatement buildSql(
             sql.append(" WHERE ")
                     .append(Joiner.on(" AND ").join(clauses));
         }
-        sql.append(appendLimitOffset(split));
-        LOGGER.debug("Generated SQL : {}", sql);
+
+        String orderByClause = extractOrderByClause(constraints);
+
+        if (!Strings.isNullOrEmpty(orderByClause)) {
+            sql.append(" ").append(orderByClause);
+        }
+
+        if (constraints.getLimit() > 0) {
+            sql.append(appendLimitOffset(split, constraints));
+        }
+
+        LOGGER.info("Generated SQL : {}", sql);
```

Comment:
this was previously a debug statement, and is now an info statement. Unless this was for a reason, let's keep it as debug.

Reply:
updated.